idea-code-style/usabilla-idea-code-style.xml

```diff
@@ -0,0 +1,48 @@
+<code_scheme name="usabilla-idea-code-style">
+  <JSCodeStyleSettings>
+    <option name="USE_SEMICOLON_AFTER_STATEMENT" value="false" />
```

Comment:
Hmm really? What's our team preference on semicolons? I thought we liked them... :)